contracts/compliance/modular/ModularCompliance.sol

```diff
@@ -198,24 +214,14 @@ contract ModularCompliance is IModularCompliance, OwnableUpgradeable, MCStorage
         assembly {
             let freeMemoryPointer := mload(0x40)
             calldatacopy(freeMemoryPointer, callData.offset, callData.length)
-            if iszero(
-            call(
-            gas(),
-            _module,
-            0,
-            freeMemoryPointer,
-            callData.length,
-            0,
-            0
-            ))
+            if iszero(call(gas(), _module, 0, freeMemoryPointer, callData.length, 0, 0))
```

Comment:
not sure it is more lisible like this, instead i would keep the formatting we had before and maybe add a comment on each line to explain what the assembly does, people don't understand assembly so it is good to document it

Reply:
could be 
```solidity
assembly {
    let freeMemoryPointer := mload(0x40) // Load the free memory pointer from memory location 0x40
    calldatacopy(freeMemoryPointer, callData.offset, callData.length) // Copy callData from calldata to the free memory location
    if iszero( // Check if the call returns zero (indicating failure)
        call( // Perform the external call
            gas(), // Provide all available gas
            _module, // Address of the target module
            0, // No ether is sent with the call
            freeMemoryPointer, // Input data starts at the free memory pointer
            callData.length, // Input data length
            0, // Output data location (not used)
            0 // Output data size (not used)
        )
    ) {
        returndatacopy(0, 0, returndatasize()) // Copy return data to memory starting at position 0
        revert(0, returndatasize()) // Revert the transaction with the return data
    }
}
```